src/Parsers/ASTSelectQuery.cpp

```diff
@@ -7,7 +7,9 @@
 #include <Parsers/ASTTablesInSelectQuery.h>
 #include <Interpreters/StorageID.h>
 #include <IO/Operators.h>
+#include <Parsers/QueryParameterVisitor.h>
 
+#include <queue>
```

Comment:
looks like this include is not needed anymore

Reply:
addressed it